config/locales/en/admin/whats_new.yml

```diff
@@ -4,14 +4,25 @@ en:
       title: What’s new in Whitehall Publisher
       summary: |
         Summary of updates to Whitehall Publisher, content design guidance, or the design of GOV.UK.
-      last_updated: Last updated 5 September 2022
+      last_updated: Last updated 3 October 2022
       introduction:
         heading: Moving to the Design System
         body_govspeak: |
           Whitehall Publisher will be moving from the current styling to use the styling and components of the [GOV.UK Design System](https://design-system.service.gov.uk/).
           This will make Whitehall Publisher more accessible, more user friendly and give you a consistent user experience throughout your publishing journey on GOV.UK.
 
           [Read more about the changes on the content community Basecamp](https://3.basecamp.com/4322319/buckets/15005645/messages/5287245970).
+      upcoming_changes:
+        heading: Upcoming changes
+        updates:
+          - heading: Topic taxonomy tags page moves to the Design System
+            area: Creating and updating documents
+            type: improved
+            date: 5 October 2022
```

Comment:
I'm wondering if having a date here is a bit confusing without context.

I interpreted the date to be the date the feature is (was) due rather than the date we posted the update.

Reply:
The date is the change is due to be released. I don't think there's a risk of confusion here based on the date being in the future. We've based this approach on other similar pages on Content Publisher and out there in industry.

Reply:
I'm confused now - I think because this date is today's date - am I understanding right from your message that we're shipping "Topic taxonomy tags" today? and putting an upcoming change that it's coming today? 😕

Reply:
Sorry @kevindew - mistake in the date which I didn't spot. It is being released tomorrow but was originally meant to be today. Now updated in a new commit.